packages/slice-machine/src/components/Breadcrumb/Breadcrumb.module.css

```diff
@@ -0,0 +1,23 @@
+.items {
+  display: flex;
+  gap: 4px;
```

Comment:
_✍️ asking for a potential change_

For both `gap: 4px`, I think we had a style issue before also. It seems in the design the gap is 4px BUT the total space is 6px. Maybe @benjammartin you could confirm what I'm saying ^^? ([see here, Breadcrumb component](https://www.figma.com/file/wb7ujCJ4j7EiP7aDDGRYXB/SliceMachine%E2%84%A2-UI?type=design&node-id=2110-267713&mode=dev))

<img width="798" alt="Screenshot 2024-04-04 at 11 09 20" src="https://github.com/prismicio/slice-machine/assets/19946868/ba780933-8932-495b-8383-b4b61a345528">

Reply:
Well spotted. It's a mistake on my part in the design. The gap is indeed 4px.

Reply:
Thank you! So no need to change anything on the code 🙂